src/main/web_src/src/components/bestilling/gjenopprett/GjenopprettGruppe.tsx

```diff
@@ -0,0 +1,77 @@
+import { TitleValue } from '~/components/ui/titleValue/TitleValue'
+import Formatters from '~/utils/DataFormatter'
+import React from 'react'
+import { GjenopprettModal } from '~/components/bestilling/gjenopprett/GjenopprettModal'
+import { DollyApi } from '~/service/Api'
+import bestilling from '~/ducks/bestilling'
+
+type GjenopprettGruppe = {
+	onCancel: any
+	gruppe: Gruppe
+	gruppeId: string
+	gruppenavn: string
+	bestillingStatuser: Record<string, BestillingStatus>
+	getBestillinger: any
+}
+
+type Gruppe = {
+	id: string
+	navn: string
+	antallIdenter: string
+}
+
+type BestillingStatus = {
+	environments: Array<string>
+}
+
+export const GjenopprettGruppe = ({
+	onCancel,
+	gruppe,
+	bestillingStatuser,
+	getBestillinger
+}: GjenopprettGruppe) => {
+	const bestilteMiljoer = () => {
+		const miljoer: Set<string> = new Set()
+		Object.values(bestillingStatuser).forEach((bestillingstatus: BestillingStatus) =>
+			bestillingstatus.environments.forEach((miljo: string) => miljoer.add(miljo))
+		)
+		return [...miljoer]
+	}
+
+	const gjenopprettHeader = (
+		<div style={{ paddingLeft: 20, paddingRight: 20 }}>
+			<h1>
+				{gruppe.navn} - #{gruppe.id}
```

Comment:
Jeg er litt fan av at hovedtittel "forklarer" modalen - kunne man kanskje satt på "Gjenopprett gruppe" før gruppenavn og id? Eller blir det for mye? 🤔

Reply:
Jeg tenkte faktisk på det samme da jeg så at gjenopprett på bestillinger heller ikke hadde noen "Gjenopprett bestilling". Kan legge det til på begge 😄

Reply:
Supert 😄